ocs_ci/utility/utils.py

```diff
@@ -1186,7 +1186,14 @@ def get_az_count():
 
 
 @retry((CephHealthException, CommandFailed), tries=20, delay=30, backoff=1)
-def ceph_health_check(namespace=None):
+def ceph_health_check(namespace=None, tries=20, delay=30, backoff=1):
```

Comment:
The arguments `tries`, `delay`, and `backoff` aren't used, can we remove them? We will also want to add `namespace` to the docstring.

Reply:
I will fix both